src/resources/projects/website/listing/item-grid.ejs.md

```diff
@@ -37,83 +37,101 @@ return !["title", "image", "image-alt", "date", "author", "subtitle", "descripti
 });
 %>
 
-<div class="g-col-1" <%= listing.utilities.metadataAttrs(item) %>>
+::: {.g-col-1 <%= listing.utilities.metadataAttrs(item) %> }
+
+```{=html}
 <a href="<%- item.path %>" class="quarto-grid-link">
-<div class="quarto-grid-item card h-100 <%-`card-${align}`%><%= hideBorders ? ' borderless' : '' %>">
+<div class="quarto-grid-item card h-100 <%- `card-${align}` %><%- hideBorders ? ' borderless' : '' %>">
```

Comment:
Switching to `<%-`.

Reply:
Mostly for clarify as to what is expected.
I'm expecting `<%=` to be further processed while `<%-` taken as is.

Reply:
> I'm expecting <%= to be further processed while <%- taken as is/escaped.

Regarding this, to avoid any confusion, 
* `<%-` escapes
* `<%=` just interpolates

This is some oddity with lodash framework which is used here, as it is reversed with other ejs framework. It was discussed previous in https://github.com/quarto-dev/quarto-cli/issues/7185#issuecomment-1850229793 

Docs for the syntax is at https://lodash.com/docs/4.17.15#template

so when you say `<%-` "as escaped", do you mean it should be escaped in output ?

Just trying to be on the same page, as this was confusing for me first time I worked on those template 😅 

> I need to do some more checks, because I personally don't like the use of that much raw inline/block.
If the div markdown syntax also avoid the memory issue, I believe it would make the EJS more readable/maintainable.µ

👍